src/core/thread/network_data.cpp

```diff
@@ -143,7 +143,7 @@ otError NetworkData::GetNextOnMeshPrefix(otNetworkDataIterator *aIterator, uint1
         aConfig->mConfigure = borderRouterEntry->IsConfigure();
         aConfig->mDefaultRoute = borderRouterEntry->IsDefaultRoute();
         aConfig->mOnMesh = borderRouterEntry->IsOnMesh();
-        aConfig->mStable = cur->IsStable();
+        aConfig->mStable = borderRouter->IsStable();
```

Comment:
If I understand correctly, we can have a situation where Prefix TLV is marked stable which contains a BorderRouter sub TLV which is not marked as stable?... @jwhui is this correct/expected behavior?

Reply:
Yes, correct.  From Thread 1.1.1 Section 5.18.2:

```
The Stable flag is set if any of the included sub-TLVs have their Stable flag set.
```

Reply:
Good. Thanks.